docs/resources/database_user_api.md

```diff
@@ -0,0 +1,85 @@
+- `labels` (Attributes List) List that contains the key-value pairs for tagging and categorizing the MongoDB database user. The labels that you define do not appear in the console. (see [below for nested schema](#nestedatt--labels))
+- `ldap_auth_type` (String) Part of the Lightweight Directory Access Protocol (LDAP) record that the database uses to authenticate this database user on the LDAP host.
+- `oidc_auth_type` (String) Human-readable label that indicates whether the new database user or group authenticates with OIDC federated authentication. To create a federated authentication user, specify the value of USER in this field. To create a federated authentication group, specify the value of IDP_GROUP in this field.
+- `password` (String) Alphanumeric string that authenticates this database user against the database specified in `databaseName`. To authenticate with SCRAM-SHA, you must specify this parameter. This parameter doesn't appear in this response.
+- `roles` (Attributes List) List that provides the pairings of one role with one applicable database. (see [below for nested schema](#nestedatt--roles))
+- `scopes` (Attributes List) List that contains clusters, MongoDB Atlas Data Lakes, and MongoDB Atlas Streams Instances that this database user can access. If omitted, MongoDB Cloud grants the database user access to all the clusters, MongoDB Atlas Data Lakes, and MongoDB Atlas Streams Instances in the project. (see [below for nested schema](#nestedatt--scopes))
+- `x509type` (String) X.509 method that MongoDB Cloud uses to authenticate the database user.
+
+- For application-managed X.509, specify `MANAGED`.
+- For self-managed X.509, specify `CUSTOMER`.
+
+Users created with the `CUSTOMER` method require a Common Name (CN) in the **username** parameter. You must create externally authenticated users on the `$external` database.
+
+### Read-Only
+
+- `links` (Attributes List) List of one or more Uniform Resource Locators (URLs) that point to API sub-resources, related API resources, or both. RFC 5988 outlines these relationships. (see [below for nested schema](#nestedatt--links))
+
+<a id="nestedatt--labels"></a>
+### Nested Schema for `labels`
+
+Optional:
+
+- `key` (String) Key applied to tag and categorize this component.
+- `value` (String) Value set to the Key applied to tag and categorize this component.
+
+
+<a id="nestedatt--roles"></a>
+### Nested Schema for `roles`
+
+Required:
+
+- `database_name` (String) Database to which the user is granted access privileges.
+- `role_name` (String) Human-readable label that identifies a group of privileges assigned to a database user. This value can either be a built-in role or a custom role.
+
+Optional:
+
+- `collection_name` (String) Collection on which this role applies.
+
+
+<a id="nestedatt--scopes"></a>
+### Nested Schema for `scopes`
+
+Required:
+
+- `name` (String) Human-readable label that identifies the cluster or MongoDB Atlas Data Lake that this database user can access.
```

Comment:
Should this be Atlas Data Lake, which is deprecated?

Reply:
Responded through slack: All these docs are grabbed directly from the API docs. If you see any relevant concerns we can create a CLOUDP to fix the API docs